contracts/WitnetRequestsBoard.sol

```diff
@@ -27,6 +27,8 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
     uint256 tallyReward;
     bytes result;
     uint256 timestamp;
+    // The epoch of the blockHash in which the last transaction related to the dr has been included
```

Comment:
If I understood it correctly, the comment could also mention that this value has different meanings in the DR lifecycle:
1. when dr is posted
2. when dr transaction is included

And then, (2) is used to check that the dr result happened after :)